### PR TITLE
fix: load when needed TestContainersConfiguration & fix broken test a…

### PR DESCRIPTION
**Pull request by vraybaud**
…fter removal of Neo4J DB #22

### CODE REVIEW

**Comment by bobeal** [comment]
recheckcla

File: entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityOperationHandlerTests.kt [hunk 1]
```diff
@@ -45,7 +49,9 @@ class EntityOperationHandlerTests {
         val jsonLdFile = ClassPathResource("/ngsild/hcmr/HCMR_test_file.json")
         val createdEntitiesIds = arrayListOf("urn:ngsi-ld:Sensor:HCMR-AQUABOX1temperature", "urn:ngsi-ld:Sensor:HCMR-AQUABOX1dissolvedOxygen", "urn:ngsi-ld:Device:HCMR-AQUABOX1")
 
-        every { entityService.exists(any()) } returns false
+        every { validationUtils.getNewEntities(any()) } returns listOf()
```
Comment (bobeal) [inline entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityOperationHandlerTests.kt:52]: you can avoid this boilerplate code by declaring the mockked bean as relaxed (`@MockkBean(relaxed = true)`), useful when you are not interested by a specific result returned by your mockked bean

**Comment by bobeal** [comment]
`EntityOperationHandlerTests` was implicitly testing (but without saying it) `ValidationUtils`. As a consequence, there are now tests in this class that are testing the exact same thing. To be dealt with during the batch creation refactoring and batch upset implementation.

File: entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityOperationHandlerTests.kt (hunk 1 above)
Comment (vraybaud) [reply entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityOperationHandlerTests.kt:52]: Thanks for the tip. :+1:

**Comment by vraybaud** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**